### PR TITLE
brush up manager example

### PR DESCRIPTION
**Pull request by sharonovd**
- bump Scylla and Scylla Manager versions
- add docker-compose
- update README accordingly

### CODE REVIEW

File: manager/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+version: "3.7"
+
+services:
+  scylla-manager:
+    image: scylladb/scylla-manager:2.6.0
```
Comment (tarzanek) [inline manager/docker-compose.yml:5]: manager should be also 2.6.3 as agent

File: manager/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      public:
+    command: --smp 1 --memory 100M
+
+  scylla-node1:
+    container_name: scylla-node1
+    build: .
+    restart: always
+    command: --seeds=scylla-node1,scylla-node2 --smp 1 --memory 750M --overprovisioned 1 --api-address 0.0.0.0
+    volumes:
+      - "./scylla/scylla.yaml:/etc/scylla/scylla.yaml"
```
Comment (tarzanek) [inline manager/docker-compose.yml:25]: is this really needed?
command should have all setup, so changing/adding scylla.yaml manually is not needed (unless you prove me wrong)
dtto for rackdc properties

File: manager/docker-compose.yml (hunk 1 above)
Comment (sharonovd) [reply manager/docker-compose.yml]: Fixed, thanks

**Review by tarzanek — APPROVED** [review]
lgtm

**Comment by tarzanek** [comment]
@guy9 from my side this can be merged (I can merge)
so whenever you say it's OK, we can just merge

**Review by guy9 — APPROVED** [review]
LGTM, thanks

**Pull request merged**